docs/src/languages/nodejs/nvm.md

```diff
@@ -17,15 +17,14 @@ variables:
     env:
         # Update these for your desired NVM and Node versions.
         NVM_VERSION: v0.35.2
```

Comment:
Maybe we can also update the `NVM_VERSION` at the same time. The latest one is `v0.36.0` now.

Reply:
Yes, why not!